index.html

```diff
@@ -12,27 +12,37 @@
 
 <body>
     <main>
+        <header>
+            <nav>
+                <a href="#headline" id="gabriel">Gabriel</a>
+                <ul class="tab tab-nav">
+                    <li class="tab-link">
+                        <a href="#portfolio">Portfolio</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#about">About</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+                <div class="hamburger ham" id="hamburger">
+                    <div class="bar-1"></div>
+                    <div class="bar-2"></div>
+                    <div class="bar-3"></div>
+                </div>
+            </nav>
+            <nav class="curtain curtain-anim" id="curtain">
+                <ul class="tab-items">
+                    <li id="nav1"><a href="#portfolio" class="mobile-links">Portfolio</a></li>
+                    <li id="nav2"><a href="#about" class="mobile-links">About</a></li>
+                    <li id="nav3"><a href="#contact" class="mobile-links">Contact</a></li>
```

Comment:
- [ ] Good job trying to implement the previous reviewer's suggestion, but it seems you have moved the menu items to the top of the page which is not making your website have a good user experience. Kindly position the menu items to match the Figma design 👍🏽  

Your design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/190711291-ad793c4d-9b19-4a50-92a8-dca7631a7bc5.png)
 
Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/190711428-d83223fa-a864-441b-b566-47867a2dd5d9.png)